packages/webapp/components/layouts/ProfileLayout/index.tsx

```diff
@@ -93,15 +84,19 @@ export default function ProfileLayout({
   );
   // Needed because sometimes initialProfile is defined and fetchedProfile is not
   const profile = fetchedProfile ?? initialProfile;
-  useEffect(() => {
-    if (profile) {
-      const purify = DOMPurify(window);
-      setTwitterHandle(sanitizeOrNull(purify, profile.twitter));
-      setGithubHandle(sanitizeOrNull(purify, profile.github));
-      setHashnodeHandle(sanitizeOrNull(purify, profile.hashnode));
-      setPortfolioLink(sanitizeOrNull(purify, profile.portfolio));
-    }
-  }, [profile]);
+  const { twitterHandle, githubHandle, hashnodeHandle, portfolioLink } =
+    useMemo(() => {
+      if (typeof window === 'undefined') {
+        return {};
+      }
```

Comment:
Maybe also just return if `profile` is undefined, no need to call `sanitizeOrNull` when stuff is undefined?

Reply:
Good point, adjusted now.